src/mmrelay/meshtastic/events.py

```
@@ -0,0 +1,978 @@
+    if not isinstance(decoded, dict):
+        decoded = {}
+
+    # Filter packets based on interaction settings
+    if decoded.get("portnum") == TEXT_MESSAGE_APP:
+        # Filter out reactions if reactions are disabled
+        if (
+            not interactions["reactions"]
+            and decoded.get("replyId") is not None
+            and "emoji" in decoded
+            and decoded.get("emoji") == EMOJI_FLAG_VALUE
+        ):
```

Ruff (0.15.9)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)